recipe/meta.yaml

```diff
@@ -55,29 +56,53 @@ requirements:
     - giflib {{ giflib }}
     - openjpeg {{ openjpeg }}
     - blosc 1.21.3
-    - lcms2 2.12
-    - libaec 1.0.4
+    - lcms2 2.16
+    - libaec 1
     - brotli 1.0.9
     - libzopfli 1.0.3
     - charls 2.2.0            # [not s390x]
     - snappy 1.1.9
-    - libtiff 4.5.0
-    - lerc 3                  # [not s390x]
+    - libtiff {{ libtiff }}
+    - lerc 4
     - zfp 1.0.0               # [not (s390x or aarch64)]
-    - libdeflate 1.17
+    - libdeflate 1.22
     - brunsli 0.1             # [not win and not s390x]
     # While jpeg-turbo is the preferred version upstream, the migration
     # to libjpeg-turbo has been contentious on conda-forge
     # - libjpeg-turbo
     - cfitsio 3.470
-    - libavif 0.11.1
+    - libavif 1.1.1
 
   run:
-    - libdeflate >=1.17,<1.18.0a0
+    - libdeflate
     - python
     # Work around missing `run_exports` in defaults package
     - "{{ pin_compatible('brotli', max_pin='x.x') }}"
-    - {{ pin_compatible('numpy') }}
+    - numpy
```

Comment:
numpy still needs to be above 1.2something and less than 3 I think, although in practice not pinning shouldn't matter too much. The upper bound might be helpful for when we update numpy to v3 (hopefully not soon)

Reply:
It's not clear what the lower bound of numpy is. They only dropped support for `Python 3.8` and `numpy < 1.21` (NEP29) in **v2023.7.4**. I'll set it in the recipe as `numpy >=1.21,<3`